Lib/test/test_audit.py

```diff
@@ -89,6 +89,10 @@ def test_excepthook(self):
         )
 
     def test_unraisablehook(self):
+        try:
+            import _testcapi
+        except ImportError:
+            raise unittest.SkipTest("requires _testcapi")
```

Comment:
Why not use `import_helper.import_module` here?

Reply:
No particular reason. I don't have a strong preference for either.

Reply:
Rewritten in a47c5ff1d7e90c5cb4bd5e74b4030251d86ee6b0